jaeger-core/src/main/java/io/jaegertracing/internal/propagation/TraceContextCodec.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
```

Comment:
2020

Reply:
we should consider removing dates altogether

Reply:
This is what the license plugin gave me

Reply:
I will have a look at this in a separate PR.